Mathlib/Analysis/InnerProductSpace/l2Space.lean

```diff
@@ -104,8 +101,6 @@ variable {G : ι → Type _} [∀ i, NormedAddCommGroup (G i)] [∀ i, InnerProd
 
 local notation "⟪" x ", " y "⟫" => @inner 𝕜 _ _ x y
 
-/-- `ℓ²(ι, 𝕜)` is the Hilbert space of square-summable functions `ι → 𝕜`, herein implemented
-as `lp (fun i : ι => 𝕜) 2`. -/
```

Comment:
Restore this?

Reply:
Yes, I'm going to do so. The rationale behind removing this, even though it fails the `docBlame` linter, is that `docBlame` is not in the CI lint.

Reply:
That's no reason to remove a helpful docstring? But, I'll be glad it's back.